website/docs/upgrade-guides.md

```diff
@@ -6,6 +6,63 @@ sidebar_label: Upgrade Guides
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+## @ionic/portals-react-native 0.1.x -> 0.2.0
+
+### API Changes
+
+#### Breaking
+
+Updated function signatures to be `async` for the following functions:
+* `register`
+* `addPortal`
+* `syncOne`
+* `syncSome`
+* `syncAll`
+
+Due to being native calls, these functions were already asynchronous. However, synchronizing these calls via `await` was impossible due the lack of exposing `async` via the API. This made all calls to these methods that didn't return a `Promise` to be fire-and-forget. These API updates allow the user to make that choice at the call-site for themselves.
+
+#### New
+
+* `addPortals` - enables adding all `Portal`s in a single call instead of having to make N calls to `addPortal` for each `Portal` to register.
+* `getPortal` - enables retrieving a `Portal` by name
```

Comment:
I actually still need to add this to the RN API 😅. I implemented this natively on both platforms but apparently forgot to add it to the JS side.